scripts/longestString.js

```diff
@@ -0,0 +1,15 @@
+function longestString(array) {
+  let lgth = 0
+  let longest;
+
+  for (var i = 0; i < array.length; i++) {
```

Comment:
you're using `let` to define variables on lines 2/3, but you're using `var` here. be sure to choose a version and stick with it 👍